content/docs/reference/version-policy.md

```diff
@@ -43,41 +43,50 @@ in terms of stability, upgradability, and the provision of services such as
 logging and monitoring. 
 
 When an application moves to version 1.0, the Kubeflow community will 
-decide whether to include that version of the application in the next major 
-or minor release of Kubeflow.
+decide whether to mark that version of the application as *stable* in the next 
+major or minor release of Kubeflow.
 
 <a id="application-matrix"></a>
 ## Kubeflow application matrix
 
-The following table shows the stable status of the applications that
-may be deployed to your Kubernetes cluster when you deploy Kubeflow. 
-The applications are specified in the 
+The following table shows the **status** (stable, beta, or experimental) of the 
+applications that you can deploy to your Kubernetes cluster when you deploy 
+Kubeflow. The applications are specified in the 
 [manifest](https://github.com/kubeflow/manifests/tree/master/kfdef) that you 
-use to deploy Kubeflow. You can use the information below to decide which of 
-the applications you should deploy to your production system.
+use to deploy Kubeflow. The kfctl deployment tool deploys the applications 
+strictly according to the manifest. kfctl does not decide whether to deploy or
+not deploy an application based on the application status.
 
-* **Stable** means that the application has reached application 
-  version 1.0 or later, and complies with the criteria defined <a href="#app-versioning">above</a> to reach version 1.0.
+You can use the information below to decide which of the applications you should
+deploy to your production system, and adjust the manifest accordingly.
+
+Application status indicators for Kubeflow:
+
+* **Stable** means that the application complies with the 
+  [criteria](https://github.com/kubeflow/community/blob/master/guidelines/application_requirements.md)
+  to reach application version 1.0, and that the Kubeflow community has deemed 
+  the application stable for this release of Kubeflow.
 * **Beta** means that the application is working towards a version 1.0 release
-  and is expected to meet the criteria for version 1.0 soon.
+  its maintainers have communicated a timeline for satisfying the criteria
```

Comment:
I think there should be an "and" before "its maintainers"? Or else the previous clause should be removed?

Reply:
Oops added the "and".